x-pack/legacy/plugins/ml/public/data_frame/common/request.ts

```diff
@@ -83,9 +83,15 @@ export function getDataFramePreviewRequest(
   groupBy: PivotGroupByConfig[],
   aggs: PivotAggsConfig[]
 ): DataFramePreviewRequest {
+  let index: string | string[] = indexPatternTitle;
```

Comment:
To make this simpler, would it be worth always making `index` a `string[]`?
We wouldn't need the comma check or index being a union type.

Reply:
We could, sure. 🤔 I can update it 👍